app/assets/javascripts/libs/webgl_stats.js

```diff
@@ -0,0 +1,337 @@
+// Adapted from https://github.com/AnalyticalGraphicsInc/webglreport
+import _ from "lodash";
+
+// eslint-disable-next-line import/prefer-default-export
+export function getWebGLReport() {
+
+  let webglVersion;
+
+  if (window.WebGLRenderingContext) webglVersion = 1;
+  if (window.WebGL2RenderingContext) webglVersion = 2;
+
+  let report = {
+      platform: navigator.platform,
+      userAgent: navigator.userAgent,
+      webglVersion
+  };
+
+
+  if ((webglVersion === 2 && !window.WebGL2RenderingContext) ||
+      (webglVersion === 1 && !window.WebGLRenderingContext)) {
+      // The browser does not support WebGL
+      return report;
```

Comment:
This if branch can never be reached, if I'm seeing this correctly. `webglVersion` is only initialized if `WebGL2RenderingContext` or `WebGLRenderingContext` are truthy. So the condition will always evaluate to false.

`if (!webglVersion) {...}` should be enough.